Dockerfile

```diff
@@ -21,14 +21,16 @@ ENV DJANGO_CONFIGURATION=${DJANGO_CONFIGURATION}
 
 # Install necessary apt packages
 RUN apt-get update && \
-    apt-get install -yq \
+    apt-get --no-install-recommends install -yq \
         software-properties-common && \
     add-apt-repository ppa:mc3man/xerus-media -y && \
     add-apt-repository ppa:mc3man/gstffmpeg-keep -y && \
     apt-get update && \
-    DEBIAN_FRONTEND=noninteractive apt-get install -yq \
-        apache2 \
+    DEBIAN_FRONTEND=noninteractive apt-get --no-install-recommends install -yq \
+        apache2 \   
         apache2-dev \
+        apt-utils \
```

Comment:
Also I think it's better not to include apt-utils to the list of installed packages, CVAT components don't depend on it. I think couple of warnings during build it's not a problem here. I tried update your PR myself, but unfortunately I could not. Otherwise PR looks good for me, all components work well. Thanks for the contribution!

Reply:
yes on one hand "CVAT" doesn't depends on "apt-utils" but on other hand i can assure it makes apt installed package configuration time small , i have seen scenarios where without "apt-utils" , it take a lot of time to configure packages after downloading , every time any new package is installed . That's why i added it here and thinks it is good to be here and i can assure it doesn't have any side effects . If u still think not to add let me know will remove it